src/shogun/solver/LDASolver.h

```diff
@@ -171,6 +171,13 @@ namespace shogun
 			    centered_class_col[c], data.get_column(i));
 			++centered_class_col[c];
 		}
+//        for (const auto& [data_i, label_i]: zip_iterator(m_features, m_labels))
```

Comment:
forgotten?

Reply:
was causing issues, so need to figure out what is happening here